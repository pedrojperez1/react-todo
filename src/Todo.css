.Todo-li {
    list-style-type: none;
}

.Todo-btn {
    margin: 10px;
}

