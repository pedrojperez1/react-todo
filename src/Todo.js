import React from "react";
import "./Todo.css"

const Todo = ({id, text, removeTodo}) => {
    const handleClick = () => {
        removeTodo(id);
    }
    return (
        <li className="Todo-li">
            {text}
            <span>
                <button className="Todo-btn" onClick={handleClick}>X</button>
            </span>
        </li>
        
    )
};

export default Todo;