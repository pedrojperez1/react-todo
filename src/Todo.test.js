import { render } from '@testing-library/react';
import Todo from './Todo';

it("should render without going up in smoke", () => {
    render(<Todo />);
});
  
it("should render the same every time", () => {
  const {asFragment} = render(<Todo />);
  expect(asFragment()).toMatchSnapshot();
});
  