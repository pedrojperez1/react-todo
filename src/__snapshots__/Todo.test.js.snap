// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the same every time 1`] = `
<DocumentFragment>
  <li
    class="Todo-li"
  >
    <span>
      <button
        class="Todo-btn"
      >
        X
      </button>
    </span>
  </li>
</DocumentFragment>
`;
